utils/generate-docs/Schema/SchemaNode/Property/InlineProperty/OneOfArray.ts

```diff
@@ -1,16 +1,17 @@
 import SchemaNode from "../../SchemaNode.js";
 import PropertyFactory, { PropertyJson } from "../Factory.js";
-import InlineProperty from "./InlineProperty.js";
+import InlineProperty, { InlinePropertyJson } from "./InlineProperty.js";
 
 interface Schema {
   findSchemaNodeById: (id: string) => SchemaNode;
 }
 
-export interface OneOfArrayJson {
```

Comment:
I had made this change when I thought we'd use a oneOf array of {"const":[obj_type]} for the `object_type` fields. Let's us dynamically change the typing in the oneOf items list. I figured I'd keep this change here as it may be useful and doesn't break anything.